platform/core/src/services/MeasurementService/index.js

```diff
@@ -0,0 +1,8 @@
+import MeasurementService from './MeasurementService';
+
+export default {
+  name: 'MeasurementService',
+  create: ({ configuration = {} }) => {
+    return new MeasurementService();
+  },
+};
```

Comment:
Ah, okay. We do this to adhere to the service pattern we created.
Almost makes me wonder if we need a simple `IService` interface that we "implement"
But I'm probably thinking too OOP

Reply:
The IService it's a good idea, at least it will make things like this more explicit and it will have a name/contract to be documented